src/core/net/ip6_mpl.hpp

```diff
@@ -216,12 +216,14 @@ class Mpl : public InstanceLocator, private NonCopyable
      * @param[in]  aMessage    A reference to the message.
      * @param[in]  aAddress    A reference to the IPv6 Source Address.
      * @param[in]  aIsOutbound TRUE if this message was locally generated, FALSE otherwise.
+     * @param[out] aReceive    Set to FALSE if the MPL message is a duplicate therefore must not
+     *                         go receiving process again, untouched otherwise.
```

Comment:
```suggestion
     * @param[out] aReceive    Set to FALSE if the MPL message is a duplicate must not
     *                         go through the receiving process again, untouched otherwise.
```